src/main/java/tc/oc/pgm/modules/ModifyBowProjectileMatchModule.java

```diff
@@ -40,44 +41,52 @@ public ModifyBowProjectileMatchModule(
 
   @EventHandler(ignoreCancelled = true)
   public void changeBowProjectile(EntityShootBowEvent event) {
-    Plugin plugin = PGM.get();
-    Entity newProjectile;
-
-    if (this.cls == Arrow.class && event.getProjectile() instanceof Arrow) {
-      // Don't change the projectile if it's an Arrow and the custom entity type is also Arrow
-      newProjectile = event.getProjectile();
-    } else {
-      // Replace the projectile
-      World world = event.getEntity().getWorld();
+    if (event.getProjectile() instanceof Arrow) {
```

Comment:
You could avoid indenting the whole method by negating this if:
`if (!(event.getProjectile() instanceof Arrow)) return;`

Reply:
I don't really like this code block in general, because the projectile in theory should always be an instanceof an arrow because the event is originating from `EntityShootBowEvent` right?